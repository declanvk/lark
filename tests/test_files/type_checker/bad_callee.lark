def foo(msg: bool,) -> bool {
    bar(msg)
    //~ ERROR: unknown identifier `bar`
}
