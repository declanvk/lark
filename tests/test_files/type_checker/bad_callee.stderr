error: unknown identifier `bar`
- bad_callee.lark:2:4
2 |     bar(msg)
  |     ^^^
